### DISCUSSION TITLE
How to get some orientations from a given alpha and beta fiber of rolled fcc

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1897] [Ask Anything]
Hi, everyone,
     I am trying to generate some orientations based on the given rolling textures.
     First I get the ebsd data of rolled Al, and want to calculate the texture strength along alpha and beta fiber, as figure shows.

     Another question is how can we extract some orientations along these two fibers? 
![image](https://github.com/mtex-toolbox/mtex/assets/69452222/4ded4d12-6f83-4716-ba1f-895fe512a96f)
![image](https://github.com/mtex-toolbox/mtex/assets/69452222/f493326c-0a89-43c1-b9e5-a32bac4fa995)

      Thanks!
      Brian

**Comment by ralfHielscher** [comment]
Use the command [`discreteSample`](https://mtex-toolbox.github.io/discreteSample.html) on the computed ODF

Ralf.
